proposals/incubation/kubeflow.md

```diff
@@ -0,0 +1,209 @@
+
+
+## Roadmap
+
+* Kubeflow - [https://github.com/kubeflow/kubeflow/blob/master/ROADMAP.md](https://github.com/kubeflow/kubeflow/blob/master/ROADMAP.md)
+* Pipelines - [https://github.com/kubeflow/pipelines/blob/master/ROADMAP.md](https://github.com/kubeflow/pipelines/blob/master/ROADMAP.md)
+* Katib - [https://github.com/kubeflow/katib/blob/master/ROADMAP.md](https://github.com/kubeflow/katib/blob/master/ROADMAP.md)
+
+## Sponsors from TOC
+
+No sponsor, yet.
+
+## Preferred maturity level
+
+Incubating
+
+## License
+
+Kubeflow is licensed under the Apache 2.0 license.
+
+## Source control
+
+[https://github.com/kubeflow](https://github.com/kubeflow) (everything in the Kubeflow org)
+
+
+## External Dependencies
+
+Kubeflow has external dependencies licensed under the following CNCF-approved licenses:
+
+* Apache 2.0
+* BSD License
+* ISC License
+* MIT License
+
+The dependency for each sub-project and their license information are included in:
+
+* Kubeflow - [https://github.com/kubeflow/kubeflow/search?q=in%3Apath+third_party+license](https://github.com/kubeflow/kubeflow/search?q=in%3Apath+third_party+license)
+* Pipelines - [https://github.com/kubeflow/pipelines/tree/master/third_party](https://github.com/kubeflow/pipelines/tree/master/third_party)
+* Training-operator - [https://github.com/kubeflow/training-operator/tree/master/third_party_licenses](https://github.com/kubeflow/training-operator/tree/master/third_party_licenses)
+* Katib - [https://github.com/kubeflow/katib/blob/master/go.mod](https://github.com/kubeflow/katib/blob/master/go.mod)
+
+When possible, sub-projects in Kubeflow will be brought in line with CNCF guidelines before submission or, in extreme circumstances, the Kubeflow project will seek CNCF governing board exceptions.
+
+## Maintainers
+
+Kubeflow welcome commits from anyone in the community. Due to its multi-repo structure with distinct contributor roles, the project uses "maintainer" as the noun for a member with sufficient permissions to merge PR in specific repo. That role can be defined at any level of each repo using the OWNERS file which in turn grants that contributor permission to “/LGTM” and “/APPROVE” PRs. These comments on PR subsequently trigger automation that actually merges that PR. 
+
+The current snapshot of the maintainers as defined above (company affiliation based on user public GitHub profile). 
+
+* Kubeflow
+    * [pdmack](https://github.com/pdmack) - Pete MacKinnon (NVIDIA)
+    * [james-jwu](https://github.com/james-jwu) - James Wu (Google)
+    * [zijianjoy](https://github.com/zijianjoy) - James Liu (Google)
```

Comment:
I think we need to add these new sections to the "maintainers" listed in the proposal:

```md
* Kubeflow Notebooks & Common Kubeflow Components
    * [elikatsis](https://github.com/elikatsis) - Ilias Katsakioris (Arrikto)
    * [kimwnasptd](https://github.com/elikatsis) - Kimonas Sotirchos (Arrikto)
    * [StefanoFioravanzo](https://github.com/StefanoFioravanzo) - Stefano Fioravanzo (Arrikto)
    * [thesuperzapper](https://github.com/thesuperzapper) - Mathew Wicks
    * [yanniszark](https://github.com/yanniszark) - Yannis Zarkadas (Arrikto)
* Kubeflow Manifests
    * [elikatsis](https://github.com/elikatsis) - Ilias Katsakioris (Arrikto)
    * [kimwnasptd](https://github.com/elikatsis) - Kimonas Sotirchos (Arrikto)
    * [PatrickXYS](https://github.com/PatrickXYS) - Yao Xiao (Google)
    * [StefanoFioravanzo](https://github.com/StefanoFioravanzo) - Stefano Fioravanzo (Arrikto)
    * [yanniszark](https://github.com/yanniszark) - Yannis Zarkadas (Arrikto)
```

Reply:
Yep, that makes sense. Will add